src/component/login/index.js

```diff
@@ -0,0 +1,180 @@
+import React, { Component } from 'react'
+import Button from '@material-ui/core/Button';
+import FormControl from '@material-ui/core/FormControl';
+import './login.css';
+import firebase from '../services/firebase.js'
+import OutlinedInput from '@material-ui/core/OutlinedInput';
+import Snackbar from '@material-ui/core/Snackbar';
+import IconButton from '@material-ui/core/IconButton';
+import Handler from '../services/userOperation';
+
+
+let database = firebase.database().ref();
```

Comment:
this line should come from a service file.

Reply:
I have removed this line from login index.js and called from service file.